.pylintrc

```diff
@@ -0,0 +1,16 @@
+[MAIN]
+ignore-paths=.*/migrations 
+
+disable=
+    line-too-long,
+    too-many-arguments,
+    too-many-locals,
+    too-many-instance-attributes,
+    too-many-return-statements,
+    fixme,
+    arguments-renamed,
+    abstract-method,
+    too-many-lines,
+    cyclic-import,
+    duplicate-code,
```

Comment:
Is this really a problem?

Reply:
It's not for most of the code but tests files are raising a lot of this error because of the way they are written. We can keep it for now but there will be more and more errors with new tests.